environments/vm/secrets/vm.yml

```diff
@@ -26,6 +26,9 @@ engine_janus_secret: secret
 
 engine_ldap_password: secret
 
+# Minimum execution time in milliseconds when response is invalid
+engine_minimum_execution_time_on_invalid_response: 5000
```

Comment:
Ah, I see. I wanted it to be configured more explicitly, I guess. But you're right: we don't need it for VM.

It would be for production to override this when necessary.